.env.development

```diff
@@ -7,3 +7,5 @@ REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
 USER_INFO_COOKIE_NAME='edx-user-info'
 SEGMENT_KEY=''
+DESIGNER_BASE_URL='http://localhost:18808'
+HOST_NAME='localhost'
```

Comment:
a bit of a nit, but I'd recommend renaming this to HOSTNAME to match this variable in other systems